src/CommunityToolkit.Maui.Core/Interfaces/IToast.shared.cs

```diff
@@ -0,0 +1,43 @@
+﻿namespace CommunityToolkit.Maui.Core;
+
+/// <summary>
+/// Toast
+/// </summary>
+public interface IToast : IAsyncDisposable
```

Comment:
Why this interface implements `IAsyncDisposable`?

Reply:
moved to IAlert. I keep Task to not break alerts and snackbar